pkg/api/context.go

```diff
@@ -26,6 +27,9 @@ import (
 // Context carries values across API boundaries.
 type Context interface {
 	Value(key interface{}) interface{}
+	Deadline() (deadline time.Time, ok bool)
```

Comment:
Is it possible to get rid of this interface since it is now identical to `context.Context`?